### ISSUE TITLE
Wallet RPC :  Mismatched txids when processing unconfimed txes from pool

### ISSUE THREAD

**Issue by bobbieltd** [MoneroOcean/nodejs-pool#6]
I run ETN pool with wallet rpc :  /usr/local/src/electroneum/bin/electroneum-wallet-rpc --rpc-bind-port 26969 --password-file ~/wallet_pass --wallet-file xxxxWallet --disable-rpc-login --trusted-daemon
It happens when payments module send ETN to miners.
I got this error . Do you know why ?
![wallet etn error](https://user-images.githubusercontent.com/29669374/34651850-b065dbdc-f408-11e7-8c91-d90756a7b53b.png)

**Comment by bobbieltd** [comment]
This issue : https://github.com/monero-project/monero/pull/2386 . It got fixed in Monero, how can I apply to Electroneum :(

**Comment by bobbieltd** [comment]
@MoneroOcean For etn.js, there is if (body.fee && body.fee > 10), do I need to change it ? Do you know what this if means and why 10 ?

**Comment by MoneroOcean** [comment]
I afraid I will not be able to help here. First of all I do not want to support ETN code and second thing is that I prefer to keep away from this sensitive part of pool code (leaving its support to Snipa). I can only guess that body.fee > 10 is just a extra check against some garbage output from wallet. Fee here is integer number that should be divided by huge 1000... to get value in XMR and it I guess Snipa decided that it can't be less than 10 under any normal circumstances. Maybe for ETN normal fees can be less than 10 here.

**Comment by bobbieltd** [comment]
Great help. Your explanation seems very correct. Thank you alot.

**Comment by MoneroOcean** [comment]
Just stumbled upon relevant commit that contains some explanations that can help you (if you still need here here of course): https://github.com/Snipa22/nodejs-pool/commit/5d31708abfc929bba4b71f8f18a5a95442b6bf91

**Comment by bobbieltd** [comment]
Thanks to your suggestions, I fixed it by myself. The ETN has only 100 atomic units, so I correct it by body.fee > 0.0000001 ( whatever number that is smaller than 0.01 should work ). At the moment, I got only issues with all rows with fee type (pool, dev, poolDev) are always zero in balance. I think pehaps because of 100 atomic units but I see all codes look fine. I can’t debug directly with my pool ( The miners are mining at pool and I’m promoting it ). In the short term, pool with 0% fee is all right. I’ll look at your fix, I think your fix is better, thank you 👍
